raiden/transfer/state.py

```diff
@@ -1783,10 +1790,3 @@ def from_dict(cls, data: Dict[str, Any]) -> 'TransactionOrder':
         )
 
         return restored
-
-
-EMPTY_MERKLE_ROOT: Locksroot = bytes(32)
-EMPTY_MERKLE_TREE = MerkleTreeState([
```

Comment:
This was a constant because the value should be treated as an immutable (even though the merkle tree has a list of lists), if anywhere in the code the merkletree was altered instead of copied things would go in havoc. This however is better done by a proper frozen class, like with a dataclasses, and since nothing is failing now just creating new objects is more than fine.